src/components/EntityInfo.vue

```diff
@@ -180,6 +197,30 @@ export default class EntityInfo extends Mixins(EnumMixin) {
     // assume Business Profile URL is always reachable
     window.location.assign(businessProfileUrl)
   }
+
+  /** Get route breadcrumbs. */
+  private getBreadcrumbs (): Array<BreadcrumbInterface> {
```

Comment:
This doesn't have to be a getter? (private get xxx(): yyy)
With the current code, does the breadcrumb update when the route changes?
Theory I'm going on here is that getters are reactive -- their value changes when the properties they depend on change -- and there's an event for whoever is using them (v-breadcrumbs).
But, if it works as-is then it works. I dunno.

Reply:
It works as is yea, it updates when the route changes. Though i think it might not need to be a computed value. 
This is me speculating, but when we change routes it is re-rendering the virtual Dom because of the changes. 
We have many instances of these functions providing dynamic values to the virtual DOM that aren't computed values, which work (i think) because Vue's reactivity system.
However, i can update this, it's no big deal!

Reply:
If it ain't broke...

Reply:
I 100% agree with you tho, i updated to a getter. It should be computed. Like all things, it made me dig a little deeper into the process and think on it!
Great convo, thanks!